src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -43,7 +43,7 @@ function ReportTypingIndicator(props) {
         case 1:
             return (
                 <TextWithEllipsis
-                    leadingText={PersonalDetails.getDisplayName(usersTyping[0])}
+                    leadingText={PersonalDetails.getDisplayNameByAccountID(usersTyping[0], 'Someone')}
```

Comment:
Although `Someone` is supposed to be a name, I think we should translate it in this case

Reply:
Definitely a good plan